src/game_constants.h

```diff
@@ -88,6 +88,18 @@
 /** Maximum range at which ranged attacks can be executed */
 #define RANGE_HARD_CAP 60
 
+/** Player view distance FLOAT */
+constexpr float player_view_distance = 60.0f;
+
+/** Player view distance INTEGER */
+constexpr int player_view_distance_int = player_view_distance;
+
+/** Distance to search for fishables from trap/fishing pole/etc */
+constexpr int fishing_distance = 60;
```

Comment:
Please hoist this to a common shared header like iuse_actor.h, no need to make this visible to the entire game.